hadoop-ozone/tools/src/main/java/org/apache/hadoop/ozone/utils/FormattingCLIUtils.java

```diff
@@ -0,0 +1,277 @@
+/**
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.apache.hadoop.ozone.utils;
+
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+/**
+ * The main core class that generates the ASCII TABLE.
+ */
+public final class FormattingCLIUtils {
```

Comment:
The comment seems a bit confusing ?

Reply:
https://github.com/apache/ozone/blob/88b88fffb1816db0c686060aa786c71ce7098138/hadoop-ozone/tools/src/main/java/org/apache/hadoop/ozone/admin/scm/GetScmRatisRolesSubcommand.java#L77-L85

Josn format is compatible with the no ratis case when `ozone.scm.ratis.enable=false`, as above. I tested in my env as follower: 
```bash
[hadoop@hadoop01 ~]$ ./scm-current/bin/ozone admin scm roles
hadoop01:9894
[hadoop@hadoop01 ~]$ ./scm-current/bin/ozone admin scm roles --json
{
  "hadoop01" : {
    "address" : "hadoop01:9894"
  }
}
```
So, it is best to also check `roles.length` on the TableFormat case to avoid NPE.

Reply:
@whbing Thanks for your suggestions! I have improved the code.